File: app/src/main/java/android/mb00/codehub/api/model/Contributor.java
package android.mb00.codehub.api.model;

import com.google.gson.annotations.SerializedName;


public class Contributor {

    @SerializedName("login")
    private String login;
    @SerializedName("avatar_url")
    private String avatarUrl;
    @SerializedName("contributions")
    private String contributions;

    public String getLogin() {
        return login;
    }

    public String getAvatarUrl() {
        return avatarUrl;
    }

    public String getContributions() {
        return contributions;
    }

}

File: app/src/main/java/mb00/android/codehub/api/model/Contributor.kt
package mb00.android.codehub.api.model

import com.google.gson.annotations.SerializedName


data class Contributor(
        @SerializedName("login") var login: String?,
        @SerializedName("avatar_url") var avatarUrl: String?,
        @SerializedName("contributions") var contributions: String?
)
